pkg/build/strategies/sti/sti.go

```diff
@@ -599,12 +607,20 @@ func (builder *STI) Execute(command string, user string, config *api.Config) err
 				continue
 			}
 			// The log level > 3 forces to use glog instead of printing to stdout
-			if glog.V(3) {
+			if glog.Is(3) {
```

Comment:
yeah but the only point of the continue statement is to avoid having an "else log to stdout".
right now the code is effectively:
if loglevel==3 {
  log to glog
} else {
  log to stdout
}

where it can just be "log to wrapper"